src/js/components/Tab/Tab.js

```diff
@@ -7,30 +7,48 @@ import { Box } from '../Box';
 import { Button } from '../Button';
 import { Text } from '../Text';
 import { normalizeColor } from '../../utils';
+import { TabsContext } from '../Tabs/TabsContext';
```

Comment:
Nit: Recommend keeping Grommet's component imports grouped alphabetically and util functions can follow.

```suggestion
import { TabsContext } from '../Tabs/TabsContext';
import { normalizeColor } from '../../utils';
```